README.md

```diff
@@ -22,12 +23,14 @@ To convert the AsciiDoc files into the resulting HTML file:
 
 See the [GitHub help](https://help.github.com/) pages and plethora of other git/GitHub guides for more details on how to work with repos, forks, pull requests, etc.
 
-### Latest Spec Build
+## Latest Spec Build
 
-Whenever a [pull request](https://github.com/cf-convention/cf-conventions/pulls) is merged, a [travis-ci build](https://travis-ci.org/github/cf-convention/cf-conventions) generates the latest specification draft and adds it to the [gh-pages branch here](https://github.com/cf-convention/cf-conventions/tree/gh-pages). The gh-pages branch is deployed to github pages at the following documents:
+Whenever a [pull request](https://github.com/cf-convention/cf-conventions/pulls) is merged, a [travis-ci build](https://travis-ci.org/github/cf-convention/cf-conventions) generates the latest specification draft and adds it to the [gh-pages branch here](https://github.com/cf-convention/cf-conventions/tree/gh-pages).
+The gh-pages branch is deployed to github pages at the following documents:
 - http://cfconventions.org/cf-conventions/cf-conventions.html
 - http://cfconventions.org/cf-conventions/cf-conventions.pdf
 - http://cfconventions.org/cf-conventions/conformance.html
 - http://cfconventions.org/cf-conventions/conformance.pdf 
 
-These documents are linked in the cf-conventions web site. [source here](https://github.com/cf-convention/cf-convention.github.io) which is published to: http://cfconventions.org/. The latest spec documents are linked from those web pages here: http://cfconventions.org/latest.html. 
\ No newline at end of file
+These documents are linked in [the cf-conventions web site](https://github.com/cf-convention/cf-convention.github.io) which is published to: http://cfconventions.org/.
```

Comment:
If we want precise URLs quoted (e.g. so printed documents state them), as I understand is the reason for not hyperlinking here (unless because it's linked closely elsewhere?), we can still hyperlink the same URL I would think, with the benefit of making it also clickable?

```suggestion
These documents are linked in [the cf-conventions web site](https://github.com/cf-convention/cf-convention.github.io) which is published to: [http://cfconventions.org/](http://cfconventions.org/).
```